src/cascadia/TerminalApp/GlobalAppSettings.h

```diff
@@ -21,6 +21,12 @@ Author(s):
 #include "Command.h"
 #include "ColorScheme.h"
 
+#ifdef _DEBUG
+static constexpr bool debugFeaturesDefault{ true };
+#else
+static constexpr bool debugFeaturesDefault{ false };

```

Comment:
please keep this in the file. it avoids leaking a global named `debugFeaturesDefault` into any file that includes GAS.h

Reply:
its location was not a mistake :smile: